e2e/fixtures/ssr-target-bundle/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ssr-target-bundle",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "waku dev --with-ssr",
+    "build": "waku build --with-ssr",
+    "start": "waku start --with-ssr"
+  },
+  "dependencies": {
+    "react": "18.3.0-canary-4b2a1115a-20240202",
+    "react-dom": "18.3.0-canary-4b2a1115a-20240202",
+    "react-server-dom-webpack": "18.3.0-canary-4b2a1115a-20240202",
+    "react-textarea-autosize": "^8.5.3",
+    "waku": "0.19.2"
```

Comment:
```suggestion
    "waku": "workspace:*"
```

Reply:
Oh I think I should change other fixtures also, because they use monorepo actually

Reply:
It's actually fine, if the version is the same, it will use the workspace version. (And, there's no way to opt-out this behavior, which is actually troublesome.)